cmd/integration/commands/state_history.go

```diff
@@ -98,3 +104,54 @@ var printCmd = &cobra.Command{
 		}
 	},
 }
+
+var rebuildCmd = &cobra.Command{
+	Use: "rebuild",
```

Comment:
no description of cli method - in `erigon snapshots --help` will be impossible to understand what `rebuild` sub-command does - because it's generic name.